src/test/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/controller/OverviewControllerTest.kt

```diff
@@ -0,0 +1,20 @@
+package uk.gov.justice.digital.hmpps.makerecalldecisionapi.controller
+
+import com.natpryce.hamkrest.assertion.assertThat
+import com.natpryce.hamkrest.equalTo
+import org.junit.jupiter.api.Test
+import uk.gov.justice.digital.hmpps.makerecalldecisionapi.domain.Crn
+import uk.gov.justice.digital.hmpps.makerecalldecisionapi.domain.FullName
+
+class OverviewControllerTest {
+  private val overviewController = OverviewController()
+
+  private val crn = Crn("12345")
+
+  @Test
+  fun `returns an overview response successfully`() {
+    val response = overviewController.overview(crn)
```

Comment:
I like the zen/minimalist test that calls the controller method directly!
A curiousity question ~ Do you use RestAssured/WebTestClient in the integration tests? Or do you tend just to keep it all minimalist like you have here?

By the way, this is a non-blocking question :)

Reply:
We've not got any real integration tests yet, so that is all up for debate. Looking at the other HMPPS projects, I can see they use WebTestClient - we can discuss it further if you want